### DISCUSSION TITLE
Which Version of JavaScript to Support

### DISCUSSION THREAD

**Discussion by apbendi** [dcgtc/dgrants#77] [Architecture & Design] [answered]
Currently configured to ES2018, but would be nice to move to ES2020 for `BigInt` support. Need to research browser support for ES2020 and decide if it's sufficient before going this route.

**Comment by phutchins** [comment, marked as answer]
After doing some digging and deep thinking, we should stay with ES2018. 

While support for BigInt has increased in various browsers, there are still many in use that do not support it. Our goal in dGrants is to enable everyone to participate in the decentralized grants platform that we're building so it makes the most sense to remain more compatible until we have a very strong reason to do otherwise.
